pages/index.js

```diff
@@ -12,6 +12,7 @@ export default function Home({ topicsData, errorCode }) {
   const { items } = useContext(LanguageContext);
   const language = items.language;
   const t = language === "en" ? en : fr;
+  const region = useRef("ON");
```

Comment:
interesting, why it works ?  good job

Reply:
> interesting, why it works ? good job

Using a reference value instead of a state allows us to update the current value (`region.current`) inside our `useEffect` without having to worry about a re-render (whereas before it would re-render every time a state was changed, whether that be setting the region after getting the geo data or setting permission status)